apps/definition_dictionary/test/dictionary/type/latitude_test.exs

```diff
@@ -2,22 +2,6 @@ defmodule Dictionary.Type.LatitudeTest do
   use ExUnit.Case
   import Checkov
 
-  describe "new/1" do
```

Comment:
Why are all these `type.new/1` tests deleted? Are we doing/testing this elsewhere now?

Reply:
Turns out these tests were never useful. They would never fail. They did a pattern match with variables, but never pinned those variables.

So we've gone quite a while with these useless tests and not once has it come up. The only reason I found it was because I expected one to fail and it didn't. 

Given their general uselessness, I deleted them all.

Reply:
Lol. Nice. Good work, Boss.